openwisp_controller/geo/tests/test_api.py

```diff
@@ -577,7 +577,7 @@ def test_change_location_type_to_outdoor_api(self):
         self._create_floorplan(location=l1)
         path = reverse('geo_api:detail_location', args=[l1.pk])
         data = {'type': 'outdoor'}
-        with self.assertNumQueries(8):
+        with self.assertNumQueries(9):
```

Comment:
The number of queries increased because of these changes https://github.com/openwisp/django-loci/pull/157